UnicornRender/video/include/unicorn/video/vulkan/Renderer.hpp

```diff
@@ -76,11 +76,12 @@ class Renderer : public video::Renderer
 {
 public:
     /**
-     * @brief Basic constructor
+     * @brief Constructor
      * @param manager Describes required extensions
      * @param window Render into
+     * @param[in] camera main camera
      */
-    Renderer(system::Manager& manager, system::Window* window);
+    Renderer(system::Manager& manager, system::Window* window, Camera& camera);
```

Comment:
I don't see any information about what happens if you pass a nullptr as window parameter. Maybe better use a reference to force not null value, or mention the behavior in the documentation?